data/tooltips.txt

```diff
@@ -536,6 +608,18 @@ tip "firing slowing / second:"
 tip "firing disruption / second:"
 	`Disrupts the ships shields after firing this weapon, allowing damage from hostile weapons to "leak through" to the hull even if shields are up. Shield disruption wears off over time.`
 
+tip "firing discharge / second:"
+	`Amount of shield damage over time by this weapon per second when firing.`
```

Comment:
Reduces shields for some time after this weapon is fired. The effect wears off over time. (also for the other texts)